### ISSUE TITLE
New Cpu Optimization issue. 

### ISSUE THREAD

**Issue by badcat109** [OBattler/86Box#128]
I seem to be having an issue when running any dos program within windows 95 osr2. upon opening it will bring emulation speed to between 0% and 5% until the dos program is closed. I was able to run a native windows game and experienced no issue. This issue presented itself with the new CPU optimization commit Build 779. 

I have tested on 2 physical machines and multiple virtual machines. 
Current setup 

Physical machines tested
 i7 6700k 4.5ghz
32gb ddr4 3200
X2 R9 390x
 Clean install Windows 10 Enterprise. 

i7 4770k 4.3ghz                                                                          
24gb ddr3 1800 ram      
X2 780ti                                                          
Clean install Windows 10 Enterprise.            

Main Test Virtual Machine config
(Windows 95OSR2)

[Machine]
model = 430vx
mem_size = 65536
cpu_use_dynarec = 1
cpu = 18
cpu_enable_fpu = 1

[Video]
gfxcard = mach64vt2
video_speed = 5
voodoo = 1

[Input devices]
mouse_type = ps2

[Sound]
sndcard = sb16
opl3_type = nukedopl

[Hard disks]
hdd_01_parameters = 63, 16, 2080, 0, ide_pio_and_dma
hdd_01_fn = C:\Users\Brad\Desktop\Win95 OSR2 ATi - Copy\Cdrive.img
hdd_01_ide_channel = 0:0

[Removable devices]
fdd_02_type = 35_2hd
cdrom_01_parameters = 1, atapi_pio_only
cdrom_01_ide_channel = 1:0
cdrom_01_host_drive = 74

[3DFX Voodoo Graphics]
type = 2
framebuffer_memory = 4
texture_memory = 4
render_threads = 2
bilinear = 1
dacfilter = 0
recompiler = 1

**Comment by OBattler** [comment]
Test if this also affects mainline PCem, if yes, report it on the PCem forum.

**Comment by OBattler** [comment]
Try the latest build of 86Box, it might be that my CPU tweak conflicted with the mainline PCem CPU optimization commit, I made my tweak only work for 486's now and DOS prompt seems to be fine in OSR 2, at least with the S3 Trio64 (I would try the Mach64 VT2 if I could find a driver for it).

**Comment by TheMadLynx** [comment]
The issue still persist: DOS programs relying on DOS/4GW shows abnormal low performance with any pentium cpu selected (especially at launch and ingame menu).

**Comment by OBattler** [comment]
Try the latest PCem compiled from the repository... comparing cutting edge 86Box which has recent mainline PCem commits applied, to PCem v12 stable which is far behind, is pointless.

**Comment by TheMadLynx** [comment]
Ok, I've tried the actual PCem from 18.07.2017 (last commit 16a453b) and it had exactly the same issue. Dreams to Reality and Pyl are games I used. Maybe there is no DOS/4GW connection -- my mistake. But performance stutters at a game launch. Also in Pyl this could be easily observed anytime you bring the menu from the actual game run. Status window shows the drop in Real speed MIPS, but effective clock remains stable.
